### PR TITLE
edit FCS_RBG_EXT to FCS_RBG

### LINKED ISSUE #260: References to FCS_RBG_EXT should be replaced with FCS_RBG
Currently there are a bunch of references to FCS_RBG_EXT SFRs in the other FCS SFRs, both in the cPP and SD. I think these should be replaced with FCS_RBG? Perhaps this will be resolved once the crypto working group sends a new version of the FCS SFRs.

### PR DESCRIPTION
**Pull request by woodbe**
cPP change for only remaining location.

SD will take a little more review since EAs have not been updated yet.

This is to close #260.

### CODE REVIEW

**Comment by jvdsn** [comment]
Is this something the crypto catalog needs to review?

**Comment by woodbe** [comment]
No, this is us just needing to clean up the move from the EXT to the CC:2022. But it isn't done yet either.

**Comment by woodbe** [comment]
I have made all the updates I was planning, this is ready for review now (hence the assignment of reviewers)

File: SD_DSC.adoc [hunk 1]
```diff
@@ -2066,18 +2066,74 @@ There are no KMD evaluation activities for this component.
...
 
+The evaluator shall verify that this function is included as an interface to the RBG in the TSS or the proprietary Entropy Analysis Report and that the behavior of the RBG following a call to this interface is described. The evaluator shall also verify that the documentation of the DRBG describes how to choose the seeding source to be used and that this source used to seed the SP 800-90A specified DRBG.
+
+====== AGD
+
+The evaluator shall verify that the operational guidance describes the process for how the seed source is selected and the output from the source supplied to the TOE's DRBG.
+
+====== Test
+
+The evaluator shall develop and execute or verify and observe the developer tooling which selects the seed source and provides it to the DRBG. The evaluator shall verify that the request succeeds.
```
Comment (jvdsn) [inline SD_DSC.adoc:2095]: "provides it" -> "provides its output"?

**Review by jvdsn — CHANGES_REQUESTED** [review]

File: SD_DSC.adoc (hunk 1 above)
Comment (woodbe) [reply SD_DSC.adoc]: edited

**Review by jvdsn — APPROVED** [review]

**Review by slpotte — APPROVED** [review]

**Pull request merged**